components/src/Form/Form.js

```diff
@@ -4,20 +4,47 @@ import { space } from "styled-system";
 import PropTypes from "prop-types";
 import {
   SectionTitle,
-  FormSection,
   Field,
   HeaderValidation,
+  Box,
+  Flex,
 } from "ComponentsRoot";
+import FormSection from "./FormSection";
 import theme from "../theme";
 
+const doNotWrapElements = [
+  FormSection,
+  HeaderValidation,
+  Box,
+  Flex,
+];
+
+const childIsWrappedByField = childType => !doNotWrapElements.includes(childType);
+
+/* eslint-disable react/no-array-index-key */
+
 const BaseForm = ({
   title,
   children,
   ...props
 }) => (
   <form { ...props }>
     <SectionTitle>{ title }</SectionTitle>
-    { children }
+    { children.map((child, index) => {
+      if (childIsWrappedByField(child.type)) {
+        return (
+          <Field key={ index }>
```

Comment:
Why does this wrapping have to happen here, as opposed to inside each form element?
I ask because `Field` is now nothing but a Box, but being used as a type.

Reply:
If teams add custom components, we can ask them to wrap them in `Field`.
It would be part of `Form`'s public API.

Thoughts?